modules/ROOT/pages/lb-cert-validation.adoc

```diff
@@ -203,37 +240,80 @@ In the command:
 * _myDlbID_ is the ID for the DLB.
 * _myAccessToken_ is the access token you generated.
 * `NewDefault` is the value of `name` for a cipher suite from the list of available cipher suites that you generated in <<List Available Cipher Suites>>.
```

Comment:
why ital instead of mono? GLOBAL

Reply:
Placeholder. Do we not use italics for placeholders?

Reply:
variables
Use monospace font for variables whether they are defined by MuleSoft or the customer:

If the variable is a place-marker for text the customer must provide, add italics to the variable name: 

`_myVar_`
``__myVar__``-plus-text-in-the-same-word

If the variable is a DataWeave variable, predefined or not, don’t add italics. 
DataWeave variables behave as functions and shouldn’t be confused with other types of variables.

If the variable is in an example block, don’t add italics (they don’t work in code blocks). You can usually indicate a customer-supplied value by naming the variable something like myVar.